packages/govuk-frontend/.browserslistrc

```diff
@@ -1,13 +1,25 @@
 # This list builds on the GOV.UK service manual's browser testing recommendations
 # https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices
 
-> 0.1%
-last 2 Chrome versions
-last 2 Firefox versions
-last 2 Edge versions
+[javascripts]
+> 0.1% in GB and supports es6-module
+last 6 Chrome versions
+last 6 Firefox versions
+last 6 Edge versions
 last 2 Samsung versions
-Safari >= 9
-iOS >= 9
+Firefox ESR
+Safari >= 11 and not Safari < 11
+iOS >= 11 and not iOS < 11
+
+[stylesheets]
+> 0.1% in GB and not dead
```

Comment:
Is there any way this can 'extend' the `javascripts` environment, rather than having to duplicate most of the list?

Reply:
Yeah, but sadly only with a separate repo or npm package. One for another day?

Reply:
Here's the info on shareable configs:
https://github.com/browserslist/browserslist#shareable-configs

Probably something we could keep in `./packages/browserslist-config-govuk-frontend` and publish one day

```suggestion
extends browserslist-config-govuk-frontend
```